packages/localize/test/LocalizeManager.test.js

```diff
@@ -593,4 +593,31 @@ describe('LocalizeManager', () => {
       );
     });
   });
+
+
+  describe('when altering html lang attribute (like Google Translate)', () => {
+    beforeEach(() => {
+      document.documentElement.lang = 'en-GB';
+    });
+
+    it("synchronizes from LocalizeManager to html[lang]", async () => {
+      manager = new LocalizeManager();
+      manager.locale = 'nl-NL';
+      expect(document.documentElement.lang).to.equal('nl-NL');
+    });
+
+    it("doesn't synchronize from html[lang] attribute to LocalizeManager initially", async () => {
+      manager = new LocalizeManager();
+      expect(manager.locale).to.equal(undefined);
+    });
+
+    it(`doesn't synchronize from html[lang] attribute to LocalizeManager
+      when changed by 3rd parties`, async () => {
+        manager = new LocalizeManager();
+        manager.locale = 'nl-NL';
+
+        document.documentElement.lang = 'fr-FR'; // this is something google translate would do
+        expect(manager.locale).to.equal('nl-NL');
```

Comment:
1. If we have the locale available, why not showing it?
2. If we don't have the locale, didn't we want to have the default locale (`en-GB`), so the translator can use that?

Btw google translate only sets the language and not the locale so `fr`

Reply:
1. it would break a11y, since end user gets a French site (translated by Google)
2. The fallback locale (en-GB) is still hardcoded in LocalizeManager
Thanks, I updated all code and docs to `fr` instead of `fr-FR`